### PR TITLE
[Improvement-16647][UI][Workflow Instance] Remove deprecated workflow execution states

### LINKED ISSUE #16647: [Improvement] [Workflow Instance] The workflow execution status has been removed from the back-end, should it not be removed from the front-end as well?
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

When I was looking at the code, I noticed the following problem:
1、 backend
image
image

2、 web
image
image

### What you expected to happen

should be removed accordingly

### How to reproduce

as shown above

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by sdhzwc**
## Purpose of the pull request
close #16647 

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

## Pull Request Notice
[Pull Request Notice](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Comment by SbloodyS** [comment]
To to target specific issue from PR, we recommend use keyword fix: #issue_id or close: #issue_id or closes: #issue_id in your PR describe(not title, just desc). It would not only connect issue to PR but also close issue automatically when PR is be closed.

**Comment by sdhzwc** [comment]
> To to target specific issue from PR, we recommend use keyword fix: #issue_id or close: #issue_id or closes: #issue_id in your PR describe(not title, just desc). It would not only connect issue to PR but also close issue automatically when PR is be closed.

image
Like this?

**Comment by SbloodyS** [comment]
> Like this?

Yes.

File: dolphinscheduler-ui/src/common/common.ts [hunk 1]
```diff
@@ -361,14 +360,6 @@ export const workflowExecutionState = (
     isSpin: false,
     classNames: 'success'
   },
-  DELAY_EXECUTION: {
-    id: 12,
-    desc: `${t('project.workflow.delay_execution')}`,
```
Comment (SbloodyS) [inline dolphinscheduler-ui/src/common/common.ts:366]: You should remove these i18n too.

Comment (sdhzwc) [reply dolphinscheduler-ui/src/common/common.ts:366]: > You should remove these i18n too.

image
image
This is referenced by Task Instance

Comment (SbloodyS) [reply dolphinscheduler-ui/src/common/common.ts:366]: The others were not used.

Comment (sdhzwc) [reply dolphinscheduler-ui/src/common/common.ts:366]: > The others were not used.

done

**Review by SbloodyS — APPROVED** [review]
LGTM

**Review by Gallardot — APPROVED** [review]

**Pull request merged**